src/packagedcode/models.py

```diff
@@ -135,153 +210,121 @@ class Package(object):
     DEPENDENCY_GROUPS = (dep_runtime, dep_dev, dep_optional, dep_test,
                          dep_build, dep_ci, dep_bundled)
 
+    # path to a file or directory
+    location = None
+
+    # the id of the package
+    id = StringType()
```

Comment:
we have an `id` in the original Pacakge class `https://github.com/nexB/scancode-toolkit/blob/develop/src/packagedcode/models.py#L152`